api/base/settings/defaults.py

```diff
@@ -61,7 +61,8 @@
 SESSION_COOKIE_SECURE = osf_settings.SECURE_MODE
 SESSION_COOKIE_HTTPONLY = osf_settings.SESSION_COOKIE_HTTPONLY
 SESSION_COOKIE_SAMESITE = osf_settings.SESSION_COOKIE_SAMESITE
-SESSION_ENGINE = 'django.contrib.sessions.backends.db'
+SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
+SESSION_CACHE_ALIAS = 'redis'
```

Comment:
What is this used for? Maybe in one: `REDIS_HOST=redis://192.168.168.167:6379`

Reply:
There are multiple cache backends defined in `CACHES` on line 339. This is to identify which backend to use.